### ISSUE TITLE
[11.0] Error on commit 3903cf5, MRP module 'o_field_pdfviewer' class does not work

### ISSUE THREAD

**Issue by uolaizola** [odoo/odoo#49223]
Impacted versions:  11.0

Steps to reproduce:

1.- Create an empty database (check Load Demonstration data)
2.- Install Manufacturing (mrp) module
3.- Go to settings -> General settings -> Manufacturing -> Check the option 'Work Orders & Quality'
3.- Go to Manufacturing menu -> Manufacturing orders -> MO/00003 -> click on 'Create Workorders'
4.- Go to Work Orders-> Open the MO/00003 - Table - Assembly Line 1 and open the 'Work instruction' tab

Current behavior:

The css rules 'o_field_pdfviewer' are not applied over the work instruction and the pdf is not correctly visualized, the image is small, the width and height are not as expected.

![pdf_mal](https://user-images.githubusercontent.com/44226087/78781837-430f6780-79a1-11ea-8fd3-4fdb35518aee.png)

Expected behavior:

The behavior previous of commit 3903cf5 where the height and witdth of 'o_field_pdfviewer' class are correctly applied and the work instruction is right. I consider that the changes of the commit are not supported in version 11.0 of Odoo. I think that it is not correct to incorporate an .scss file with sass format that is not supported, it causes and error and without assets the rules of the mrp.less file are not applied on the web browser

![pdf_bien](https://user-images.githubusercontent.com/44226087/78781896-57ebfb00-79a1-11ea-9f08-3089d19e896f.png)

Video/Screenshot link (optional):

**Comment by pedrobaeza** [comment]
Is this reproducible on runbot?

**Comment by uolaizola** [comment]
Yes, sure, it is reproducible on runbot, here it is a videoscreen.


[2020-04-08 15-03-36.zip](https://github.com/odoo/odoo/files/4450656/2020-04-08.15-03-36.zip)

**Comment by jhk-odoo** [comment]
Hello, this looks indeed like unexpected behavior. If you can provide me with the opw id, i'll add this ticket to my list.

**Comment by uolaizola** [comment]
Hello,

Please could you tell me how can I get the opw id ? We are using an 'Odoo 11.0 (Community Edition) '

Thanks in advance

**Comment by pedrobaeza** [comment]
That's only for enterprise customers.

**Comment by uolaizola** [comment]
Thank you so much Pedro. Therefore, do I need to do something else in order to the fix of this issue goes on ?

**Comment by pedrobaeza** [comment]
Well, that's not on my side, as I'm a voluntary, nor Odoo, as they don't guarantee the bug fixing unless you have an enterprise contract, which is reasonable, and this only relies on the voluntary effort of someone (from Odoo or community) bringing the patch.

**Comment by jhk-odoo** [comment]
i'll keep this in mind for the next time i'm in between tickets.

**Comment by uolaizola** [comment]
Thanks to both of you for your help.

**Comment by pedrobaeza** [comment]
@jhk-odoo were you able to fix this?

**Comment by jhk-odoo** [comment]
it's on my list for this week
opw-2254607

**Comment by pedrobaeza** [comment]
Thanks!
